javascript/ditsmod/package.json

```diff
@@ -15,9 +15,9 @@
   "author": "Your Name Here",
   "license": "MIT",
   "dependencies": {
-    "@ditsmod/body-parser": "~2.17.1",
-    "@ditsmod/core": "~2.61.1",
-    "@ditsmod/routing": "~2.7.0"
+    "@ditsmod/body-parser": "^3.0.0-alpha.2",
+    "@ditsmod/core": "^3.0.0-alpha.2",
```

Comment:
When do you plan to release a non beta / alpha ?

Reply:
No plan, for now =).

Reply:
Not sure it's a good idea to use beta/alpha, but policy is not clear enough for that.

Will accept this for now

Reply:
Thank you. Do not doubt that when a stable version is released, I will definitely update it on your project as well.